src/Desktop.Analyzers/Desktop.Analyzers.md

```diff
@@ -48,6 +48,16 @@ Severity: Warning
 
 Help: [http://msdn.microsoft.com/library/ms182350.aspx](http://msdn.microsoft.com/library/ms182350.aspx)
```

Comment:
Oh! I did't know this came from a tool and that the original strings were in code. In that case I withdraw the suggestion. And in that case, yes, these are going to be unformatted paragraphs of code, so ignore what I said later about formatting.